docs/jump/README.md

```diff
@@ -0,0 +1,71 @@
+![Logo](https://raw.githubusercontent.com/infiniteopt/InfiniteOpt.jl/master/full_logo.png)
+
+A `JuMP` extension for expressing and solving infinite-dimensional optimization
+problems. Such areas include [stochastic programming](https://en.wikipedia.org/wiki/Stochastic_programming),
+[dynamic programming](https://en.wikipedia.org/wiki/Dynamic_programming),
+space-time optimization, and more. `InfiniteOpt` serves as an easy-to-use modeling
+interface for these advanced problem types that can be used by those with little
+background in these areas. It also it contains a wealth of capabilities
+making it a powerful and convenient tool for advanced users.  
+
+| **Current Version**                     | **Documentation**                                                               | **Build Status**                                                                                | **Citation** |
+|:---------------------------------------:|:-------------------------------------------------------------------------------:|:-----------------------------------------------------------------------------------------------:|:--------------------------------------:|
+| [![](https://docs.juliahub.com/InfiniteOpt/version.svg)](https://juliahub.com/ui/Packages/InfiniteOpt/p3GvY) | [![](https://img.shields.io/badge/docs-stable-blue.svg)](https://infiniteopt.github.io/InfiniteOpt.jl/stable) | [![Build Status](https://github.com/infiniteopt/InfiniteOpt.jl/workflows/CI/badge.svg?branch=master)](https://github.com/infiniteopt/InfiniteOpt.jl/actions?query=workflow%3ACI) [![codecov.io](https://codecov.io/github/infiniteopt/InfiniteOpt.jl/coverage.svg?branch=master)](https://codecov.io/github/infiniteopt/InfiniteOpt.jl?branch=master) | [![DOI](https://img.shields.io/badge/Elsevier-CompChemEng%3A107567-yellow.svg)](https://doi.org/10.1016/j.compchemeng.2021.107567) |
+
+`InfiniteOpt` builds upon `JuMP` to add support for many complex modeling objects which 
+include:
+- Infinite parameters (e.g., time, space, uncertainty, etc.)
+- Finite parameters (similar to `ParameterJuMP`)
+- Infinite variables (decision functions) (e.g., `y(t, x)`)
+- Derivatives (e.g., `∂y(t, x)/∂t`)
+- Measures (e.g., `∫y(t,x)dt`, `𝔼[y(ξ)]`)
+- 1st class nonlinear modeling
+
+The unifying modeling abstraction behind `InfiniteOpt` captures a wide spectrum 
+of disciplines which include dynamic, PDE, stochastic, and semi-infinite 
+optimization. Moreover, we facilitate transferring techniques between these 
+to synthesize new optimization paradigms!
+
+## License
+`InfiniteOpt` is licensed under the [MIT "Expat" license](./LICENSE).
+
+## Installation
+`InfiniteOpt.jl` is a registered [Julia](https://julialang.org/) package and 
+can be installed by entering the following in the REPL.
+
+```julia
+julia> ]
```

Comment:
I think we should prefer `import Pkg; Pkg.add("InfiniteOpt")`. I've found that many users get confused with this because they try copy-pasting:
```Julia
julia> ]
ERROR: syntax: unexpected "]"
Stacktrace:
 [1] top-level scope
   @ none:1
```
It only work if you _type_ the bracket.

Reply:
Ok I can make a new pull request for this,